### PR TITLE
[MGDSTRM-9160] Process authentication and admin client construction as blocking

### PR DESCRIPTION
**Pull request by MikeEdgar**
Avoids complexity of running various subtasks of some operations on a worker thread.

### CODE REVIEW

File: kafka-admin/src/main/java/org/bf2/admin/kafka/admin/handlers/RestOperations.java [hunk 1]
```diff
@@ -37,6 +37,7 @@
 import java.util.stream.Collectors;
 
 @Path("/api/v1")
+@Blocking
```
Comment (vbusch) [inline kafka-admin/src/main/java/org/bf2/admin/kafka/admin/handlers/RestOperations.java:40]: @MikeEdgar is there a performance impact here that we should test for?

Comment (MikeEdgar) [reply kafka-admin/src/main/java/org/bf2/admin/kafka/admin/handlers/RestOperations.java:40]: I'm looking into this.

Comment (MikeEdgar) [reply kafka-admin/src/main/java/org/bf2/admin/kafka/admin/handlers/RestOperations.java]: I ended up backing out this change and adjusting the `createAdminClient` method to return a `CompletableFuture` where the Kafka client creation occurs in on worker thread. Also the time allowed for processes running on an event loop thread was increased from the default of 2000ms to 4000ms. This seems reasonable for situations where the server is running with constrained CPU.

I was also successful in getting this to run with a native executable (a few other small changes) with very good performance.

**Review by rareddy — APPROVED** [review]
lgtm

**Review by vbusch — APPROVED** [review]
LGTM

**Pull request merged**